### PR TITLE
Update Output docs

### PR DESCRIPTION
**Pull request by setrofim**
Plus a minor adjustment to JobOutput's __str__.

### CODE REVIEW

File: doc/source/user_guide.rst [hunk 1]
```diff
@@ -435,32 +435,59 @@ To display verbose output while running memcpy::
...
-running on a platform that supports android then WA will take a screen capture
-and UI dump from the device.
-
-Finally, there will be a ``__meta`` subdirectory. This will contain a copy of
-the agenda file used to run the workloads along with other configuration
-files that were used for the execution of that run.
+Output
+======
+
+The output directory will contain, subdirectories for each job that was run,
```
Comment (Sticklyman1936) [inline doc/source/user_guide.rst]: Erroneous comma

File: doc/source/user_guide.rst [hunk 2]
```diff
@@ -435,32 +435,59 @@ To display verbose output while running memcpy::
...
+which will in turn contain the generated metrics and artificats for each job.
```
Comment (Sticklyman1936) [inline doc/source/user_guide.rst]: Art[ei]facts

File: doc/source/user_reference/output_directory.rst [hunk 3]
```diff
@@ -0,0 +1,132 @@
+.. _output_directory_structure:
+
+Output Directory Structure
+==========================
+
+This is an over overview of WA output directory structure.
```
Comment (Sticklyman1936) [inline doc/source/user_reference/output_directory.rst]: Do you mean just "overview"? Or maybe "high-level overview"?

File: doc/source/user_reference/output_directory.rst [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+.. note:: In addition to files and subdirectories described here,
+          other content may present int the output directory for
+	  a run, depending on the enabled augmentations.
```
Comment (Sticklyman1936) [inline doc/source/user_reference/output_directory.rst]: Incorrect indentation

File: doc/source/user_reference/output_directory.rst [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+	├── wk2-memcpy-1/
+	│   └── result.json
+	├── wk2-memcpy-2/
+	│   └── result.json
+	├── result.json
+	└── run.log
+
+This is the directory structure that would be generated after running two
+iterations each of ``dhrystone`` and ``memcpy`` workloads with no augmentations
+enabled, and with the first attempt of the first iteration of dhrystone having
```
Comment (Sticklyman1936) [inline doc/source/user_reference/output_directory.rst]: at the first iteration

File: doc/source/user_guide.rst [hunk 6]
```diff
@@ -435,32 +435,59 @@ To display verbose output while running memcpy::
...
+augmaintations may add additional files at the run or job directory level. The
+default configuration has ``status`` and ``csv`` augmentations enabled which
+genrate a ``status.txt`` containing status summary for the run and individual
+jobs, and a ``results.csv`` containging metrics from all jobs in a CSV table,
+respectively.
+
+See :ref:`output_directory_structure` for more information.
+
+In order to make it easier to access WA results from scripts, WA provides an API
+that prases the contents of the output directory:
```
[the remark this answers is not in the document]
Comment (Sticklyman1936) [reply doc/source/user_guide.rst]: parses!

File: doc/source/user_guide.rst [hunk 7]
```diff
@@ -435,32 +435,59 @@ To display verbose output while running memcpy::
...
+
+
+.. code-block:: pycon
```
[the remark this answers is not in the document]
Comment (setrofim) [reply doc/source/user_guide.rst]: "pycon" is correct; stands for "python console". See:
http://pygments.org/docs/lexers/#lexers-for-python-and-related-languages

File: doc/source/user_guide.rst (hunk 2 above)
Comment (setrofim) [reply doc/source/user_guide.rst]: artifacts  -- the US spelling is used throughout.

File: doc/source/user_reference/output_directory.rst (hunk 3 above)
Comment (setrofim) [reply doc/source/user_reference/output_directory.rst]: Erm, what's the difference in this context?

File: doc/source/user_reference/output_directory.rst (hunk 5 above)
Comment (setrofim) [reply doc/source/user_reference/output_directory.rst]: I believe "of" is correct here; "at" precedes a verb:

https://www.englishforums.com/English/AttemptToVsAttemptAt/wzrwh/post.htm

Comment (setrofim) [reply doc/source/user_reference/output_directory.rst]: Erm, never mind, I got that exactly the wrong way round.

**Comment by setrofim** [comment]
Comments addressed.

**Pull request merged**